apps/ledger-live-desktop/src/newArch/features/ModularDrawer/components/SelectAccountFlow/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { memo, useState } from "react";
+import { Observable } from "rxjs";
+import { useTranslation } from "react-i18next";
+import { CryptoOrTokenCurrency } from "@ledgerhq/types-cryptoassets";
+import { Account, AccountLike } from "@ledgerhq/types-live";
+import { WalletAPIAccount } from "@ledgerhq/live-common/wallet-api/types";
+import { SelectAccountFlowContainer, SelectorContent, AccountSelectionStep } from "./components";
+import MemoizedSelectAssetFlow from "../SelectAssetFlow";
```

Comment:
IMO there's no need to indicate that SelectAssetFlow is memoized:
```suggestion
import SelectAssetFlow from "../SelectAssetFlow";
```